Lesson Modules/Module 1- Import Local CSVs.ipynb

```diff
@@ -29,20 +29,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Normally, your file of interest is not located in the default working directory of Python. The command os.chdir() changes the working directory so we can open our file in this script."
+    "Normally, you're file of interest is not located in the default working directory of Python. os.chdir() changes the working directory so we can open our file of interest."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
    "outputs": [],
    "source": [
-    "filepath = 'C:/Users/Radley/Downloads'   # save file location as a string.\n",
-    "os.chdir(filepath)                       # and use os.chdir() to change the directory\n",
-    "filename = 'lbl.csv'                     # now that you are in the correct directory, store the file name"
+    "filepath = 'C:/Users/Radley/Downloads/' # store file location as a string and be sure to end filepath with '/'\n",
+    "filename = 'lbl.csv'                    # store the file name as a string\n",
+    "os.chdir(filepath)                      # and use os.chdir() to change the directory\n"
```

Comment:
You don't need this `os.chdir` anymore because you include `filepath` in the arguments for the functions.